### PR TITLE
Peizer branch

### PR DESCRIPTION
**Pull request by bpeizer**

### CODE REVIEW

File: JenkinsFile [hunk 1]
```diff
@@ -11,7 +11,7 @@ node {
     git([
       url: "https://github.com/venicegeo/bftest-integration",
       poll: true,
-      branch: "master"
+      branch: "PeizerBranch"
```
Comment (jamesyarrington) [inline JenkinsFile]: You should make sure everything references "master", once this is pushed to master.

File: ci/Postman/beachfront.sh [hunk 2]
```diff
@@ -42,18 +42,39 @@ spaces=int
 for space in $spaces; do
 	# Reinitialize "latch" for the tests against the current space.
 	latch=0
-	
+	case $space in
```
Comment (jamesyarrington) [inline ci/Postman/beachfront.sh:45]: are int/test/stage/prod still the names of the environments?

File: ci/Selenium/src/test/java/bfui/test/util/Utils.java [hunk 3]
```diff
@@ -335,7 +335,7 @@ public static void takeSnapShot(WebDriver webdriver, String fileName) throws Exc
 
 	            //Move image file to new destination
 
-	                File DestFile=new File("/Users/Peizer/Documents/Old/bftest-integration/ci/Selenium/"+fileName);
+	                File DestFile=new File("/Users/Peizer/Documents/bftest-integration/ci/Selenium/"+fileName);
```
Comment (jamesyarrington) [inline ci/Selenium/src/test/java/bfui/test/util/Utils.java]: Does this path work in Jenkins?  Or is this test only to be run on your machine?

File: ci/Selenium/src/test/java/bfui/test/TestJobsList.java [hunk 4]
```diff
@@ -122,13 +126,18 @@ public void view_on_map() {
 	}
 	
 	@Test @Info(importance = Importance.HIGH)
-	public void download_geojson_result() throws InterruptedException {
+	public void download_geojson_result() throws InterruptedException, IOException {
 		// Make sure that the "Download" Job button does something.  Selenium cannot tell if a download occurred.
 		//assertEquals("There should not be a download link before clicking", null, testJob.downloadLink.getAttribute("href"));
+		String home = System.getProperty("user.home");
+		File file = new File(home+"/Downloads/"+"ForJobTesting"+".geojson");
```
Comment (jamesyarrington) [inline ci/Selenium/src/test/java/bfui/test/TestJobsList.java:133]: Does this path work on the machine doing the testing?

File: ci/Selenium/src/test/java/bfui/test/TestJobsList.java [hunk 5]
```diff
@@ -143,14 +152,22 @@ public void download_geojson_result() throws InterruptedException {
...
+		Assert.assertTrue("File Size is larger than 1kb",file.length()>1000);
 			//Assert.assertTrue("Download path should appear after click", testJob.downloadLink.getAttribute("href").contains("blob"));
 	}
 	@Test @Info(importance = Importance.HIGH)
-	public void download_geopackage_result() throws InterruptedException {
+	public void download_geopackage_result() throws InterruptedException, IOException {
 		// Make sure that the "Download" Job button does something.  Selenium cannot tell if a download occurred.
 		//assertEquals("There should not be a download link before clicking", null, testJob.downloadLink.getAttribute("href"));
+		String home = System.getProperty("user.home");
+		File file = new File(home+"/Downloads/"+"ForJobTesting"+".gpkg");
```
Comment (jamesyarrington) [inline ci/Selenium/src/test/java/bfui/test/TestJobsList.java:166]: Does this path work on the machine doing the testing?

File: ci/Selenium/src/test/java/bfui/test/TestJobsList.java [hunk 6]
```diff
@@ -168,9 +185,45 @@ public void download_geopackage_result() throws InterruptedException {
...
+		//Thread.sleep(2000);
+		//Assert.assertTrue("Download path should appear after click", testJob.downloadLink.getAttribute("href").contains("blob"));
+	}
+	
+	@Test @Info(importance = Importance.HIGH)
+	public void download_shapefile_result() throws InterruptedException, IOException {
+		// Make sure that the "Download" Job button does something.  Selenium cannot tell if a download occurred.
+		//assertEquals("There should not be a download link before clicking", null, testJob.downloadLink.getAttribute("href"));
+		String home = System.getProperty("user.home");
+		File file = new File(home+"/Downloads/"+"ForJobTesting"+".shp.zip");
```
Comment (jamesyarrington) [inline ci/Selenium/src/test/java/bfui/test/TestJobsList.java:201]: Does this path work on the machine doing the testing?

File: ci/Postman/beachfront.sh (hunk 2 above)
Comment (bpeizer) [reply ci/Postman/beachfront.sh:45]: technically the spaces are pz-int pz-test and pz-prod but haveing them as int test and prod in the postman test doesnt seem to be causing an issue. I can go back and refactor if you would like.

File: ci/Selenium/src/test/java/bfui/test/util/Utils.java (hunk 3 above)
Comment (bpeizer) [reply ci/Selenium/src/test/java/bfui/test/util/Utils.java]: This section is only to be used for local test debugging, images wont be able to be saved via jenkins so the user would need to update the path for where they want to see the image.

File: ci/Selenium/src/test/java/bfui/test/TestJobsList.java (hunk 4 above)
Comment (bpeizer) [reply ci/Selenium/src/test/java/bfui/test/TestJobsList.java:133]: yes

File: ci/Selenium/src/test/java/bfui/test/TestJobsList.java (hunk 5 above)
Comment (bpeizer) [reply ci/Selenium/src/test/java/bfui/test/TestJobsList.java:166]: yes

File: ci/Selenium/src/test/java/bfui/test/TestJobsList.java (hunk 6 above)
Comment (bpeizer) [reply ci/Selenium/src/test/java/bfui/test/TestJobsList.java:201]: yes

File: ci/Postman/beachfront.sh (hunk 2 above)
Comment (jamesyarrington) [reply ci/Postman/beachfront.sh:45]: no, using those names should be fine

File: ci/Selenium/src/test/java/bfui/test/util/Utils.java (hunk 3 above)
Comment (jamesyarrington) [reply ci/Selenium/src/test/java/bfui/test/util/Utils.java]: Add that as a comment here, for future reference.

**Pull request merged**